src/workerd/io/worker.c++

```diff
@@ -2140,6 +2161,58 @@ public:
 private:
   kj::WebSocket& webSocket;
 
+  void takeHeapSnapshot(jsg::Lock& js, bool exposeInternals, bool captureNumericValue) {
+    struct Activity: public v8::ActivityControl {
+      InspectorChannelImpl& channel;
+      Activity(InspectorChannelImpl& channel) : channel(channel) {}
+
+      ControlOption ReportProgressValue(uint32_t done, uint32_t total) {
+        capnp::MallocMessageBuilder message;
+        auto event = message.initRoot<cdp::Event>();
+        auto params = event.initReportHeapSnapshotProgress();
+        params.setDone(done);
+        params.setTotal(total);
+        if (done == total) {
+          params.setFinished(true);
+        }
+        auto notification = getCdpJsonCodec().encode(event);
+        channel.sendNotification(kj::mv(notification));
+        return ControlOption::kContinue;
+      }
+    };
+
+    struct Writer: public v8::OutputStream {
+      InspectorChannelImpl& channel;
+
+      Writer(InspectorChannelImpl& channel) : channel(channel) {}
+      void EndOfStream() override {}
+
+      int GetChunkSize() override {
+        return 65536;  // big chunks == faster
```

Comment:
```suggestion
        return 65536;  // big chunks == faster
        // The chunk size here will determine the actual number of individual
        // messages that are sent. The default is... rather small. Experience
        // node and node-heapdump shows that this can be bumped up
        // much higher to get better performance. Here we use the value
        // that Node.js uses (see Node.js' FileOutputStream impl).
```

Reply:
It looks like you removed this comment in a later update -- was that intentional?

Reply:
Aw crap, no that was a merge glitch. Thanks for catching that, I'll add it back